hops/serving.py

```diff
@@ -378,7 +429,67 @@ def get_artifact_path(serving_name):
     """
     servings = get_all()
     serving = _find_serving_with_name(serving_name, servings)
-    return serving.artifact_path
+    return serving.model_path
+
+
+def get_model_version(serving_name):
+    """
+    Gets the version of the model served by a given serving instance
+
+    Example use-case:
+
+    >>> from hops import serving
+    >>> serving.get_model_version(serving_name)
+
+    Args:
+        :serving_name: name of the serving to get the version for
+
+    Returns:
+         the version of the serving
```

Comment:
The version of the model being served right?

Reply:
Fixed